src/chart/ui/events.cpp

```diff
@@ -30,7 +30,7 @@ std::string PointerEvent::dataToJson() const
 		coords = chart->getCoordSystem().getOriginal(position);
 	}
 	return "\"element\":\"" + elementUnder + "\""
-	     + ",\"pointerId\":" + std::to_string(pointerId)
+	     + ",\"pointerId\":" + std::to_string(pointerId.value_or(-1))
```

Comment:
here would be better to put null to json instead of -1 in case of not set

Reply:
I replaced to Conv::toString(pointerId).